include/Physics/Map.h

```diff
@@ -0,0 +1,28 @@
+//
+// Created by SpiritStudio on 28.11.18.
+//
+
+#ifndef VEHICLESEVOLUTION_MAP_H
+#define VEHICLESEVOLUTION_MAP_H
+
+#include <Box2D/Box2D.h>
+
+class Map {
+
+public:
+    Map(b2World &world);
+
+    b2Vec2 getPosition() const;
+    double getAngle() const;
+    std::vector<b2Vec2> getPolyline() const;
```

Comment:
Here it is actually possible, because `std::vector<b2Vec2> polyline_` exists (https://github.com/SpiritStudio/VehiclesEvolution/pull/17#discussion_r243749942)